src/main/java/com/avispl/symphony/dal/avdevices/bridges/vaddio/avbridgenano/VaddioBridgeNanoCommunicator.java

```diff
@@ -0,0 +1,1045 @@
+import com.avispl.symphony.dal.avdevices.bridges.vaddio.avbridgenano.common.StreamSettings;
+import com.avispl.symphony.dal.avdevices.bridges.vaddio.avbridgenano.common.VaddioCommand;
+import com.avispl.symphony.dal.avdevices.bridges.vaddio.avbridgenano.common.VaddioNanoConstant;
+import com.avispl.symphony.dal.communicator.SshCommunicator;
+import com.avispl.symphony.dal.util.StringUtils;
+
+/**
+ * VaddioBridgeNanoCommunicator An implementation of SshCommunicator to provide communication and interaction with Vaddio Bridge Nano device
+ *
+ * Monitoring
+ * Network information
+ * Streaming Settings
+ *
+ * Controlling
+ * Mute
+ * Volume(dB)
+ * Gain(dB)
+ * Reboot
+ * Video Mute
+ * Audio Mute
+ *
+ * @author Kevin / Symphony Dev Team<br>
+ * Created on 8/14/2023
+ * @since 1.0.0
+ */
+public class VaddioBridgeNanoCommunicator extends SshCommunicator implements Monitorable, Controller {
+
+	/**
+	 * cache to store key and value
+	 */
+	private final Map<String, String> cacheKeyAndValue = new HashMap<>();
+
+	/**
+	 * count the failed command
+	 */
+	private final Map<String, String> failedMonitor = new HashMap<>();
+
+	/**
+	 * Prevent case where {@link VaddioBridgeNanoCommunicator#controlProperty(ControllableProperty)} slow down -
+	 * the getMultipleStatistics interval if it's fail to doGet the cmd
+	 */
+	private static final int controlSSHTimeout = 3000;
+	/**
+	 * Set back to default timeout value in {@link SshCommunicator}
+	 */
+	private static final int statisticsSSHTimeout = 30000;
+
+	/**
+	 * ReentrantLock to prevent telnet session is closed when adapter is retrieving statistics from the device.
+	 */
+	private final ReentrantLock reentrantLock = new ReentrantLock();
+
+	/**
+	 * Store previous/current ExtendedStatistics
+	 */
+	private ExtendedStatistics localExtendedStatistics;
+
+	/**
+	 * configManagement imported from the user interface
+	 */
+	private String configManagement;
+
+	/**
+	 * configManagement in boolean value
+	 */
+	private boolean isConfigManagement;
+	private boolean isEmergencyDelivery;
+	private boolean isNextPollingInterval;
+
+	/**
+	 * Retrieves {@link #configManagement}
+	 *
+	 * @return value of {@link #configManagement}
+	 */
+	public String getConfigManagement() {
+		return configManagement;
+	}
+
+	/**
+	 * Sets {@link #configManagement} value
+	 *
+	 * @param configManagement new value of {@link #configManagement}
+	 */
+	public void setConfigManagement(String configManagement) {
+		this.configManagement = configManagement;
+	}
+
+	/**
+	 * Constructor for VaddioBridgeNanoCommunicator class
+	 */
+	public VaddioBridgeNanoCommunicator() {
+		this.setCommandErrorList(Collections.singletonList("Error: response error"));
+		this.setCommandSuccessList(Collections.singletonList("> "));
+		this.setLoginSuccessList(Collections.singletonList("********************************************\r\n        \r\nWelcome admin\r\n> "));
```

Comment:
is it always this welcome message? I just wonder if it's this specific, or the amount of spaces/line breaks is different. 
Also, I'm pretty sure the response lists are considering asterisks as "any character", so probably there's no need to write all of them, but instead "*Welcome admin*>" or something would do. Can you please check if this is the case? 

Another one - is it always "Welcome **admin**", regardless of the username? Just wonder if it's based on the username, so if the username is, let's say, **mike** - it will respond with the welcome message "Welcome mike", thus ending up with a Login Error